File: Python/008_StringScore.py
# https://leetcode.com/problems/score-of-a-string/description/


def solution(s):
    sum = 0
    for i in range(1, len(s)):
        subsum = abs(ord(s[i]) - ord(s[i - 1]))
        sum += subsum
    print(sum)


s = "hello"
solution(s)

File: C Sharp/008_StringScore.cs
    public int ScoreOfString(string s) {
        int sum = 0;
        for(int i = 1;i < s.Length ; i++){
            sum += Math.Abs((int)s[i-1] - (int)s[i]);
        }

        return sum;
    }


    // # https://leetcode.com/problems/score-of-a-string/description/
